deps/curl.mk

```diff
@@ -0,0 +1,46 @@
+## CURL ##
+
+CURL_SRC_TARGET := $(BUILDDIR)/curl-$(CURL_VER)/.libs/libcurl.$(SHLIB_EXT)
+CURL_OBJ_TARGET := $(build_shlibdir)/libcurl.$(SHLIB_EXT)
+
+ifneq ($(OS),WINNT)
+CURL_LDFLAGS := "-Wl,-rpath,'$(build_libdir)'"
+endif
+
+$(SRCDIR)/srccache/curl-$(CURL_VER).tar.bz2: | $(SRCDIR)/srccache
+	$(JLDOWNLOAD) $@ https://curl.haxx.se/download/curl-$(CURL_VER).tar.bz2
+$(SRCDIR)/srccache/curl-$(CURL_VER)/configure: $(SRCDIR)/srccache/curl-$(CURL_VER).tar.bz2
+	$(JLCHECKSUM) $<
+	cd $(dir $<) && $(TAR) jxf $(notdir $<)
+	touch -c $@
+$(BUILDDIR)/curl-$(CURL_VER)/config.status: $(SRCDIR)/srccache/curl-$(CURL_VER)/configure
+	mkdir -p $(dir $@)
+	cd $(dir $@) && \
+	$< $(CONFIGURE_COMMON) --includedir=$(build_includedir) --with-mbedtls=$(build_shlibdir) CFLAGS="$(CFLAGS) $(CURL_CFLAGS)" LDFLAGS="$(LDFLAGS) $(CURL_LDFLAGS)"
```

Comment:
I think you may need to also do `--disable-ssl` or something like that to get it to not link against openssl?